docs/template_plugin/tests/functional/op_reference/reduce_l2.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+
+#include "reduction.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using ReductionType = ngraph::helpers::ReductionType;
+
+namespace reference_tests {
+namespace ReductionOpsRefTestDefinitions {
+namespace {
+
+std::vector<ReductionParams> reductionParams = {
+    ReductionParams(PartialShape{3, 2, 2}, element::f32,
+                    std::vector<float>{1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0},
+                    std::vector<int64_t>{2},
+                    std::vector<float>{2.23606798, 5.0, 7.81024968, 10.63014581, 13.45362405, 16.2788206},
+                    true,
+                    ReductionType::L2),
+    ReductionParams(PartialShape{3, 2, 2}, element::f32,
```

Comment:
Could we cover all supported data types?

Reply:
Done